py/server/deephaven/table.py

```diff
@@ -1244,15 +1244,22 @@ def format_row_where(self, cond: str, formula: str) -> Table:
             raise DHError(e, "failed to color format rows conditionally.") from e
 
     def layout_hints(self, front: Union[str, List[str]] = None, back: Union[str, List[str]] = None,
-                     freeze: Union[str, List[str]] = None, hide: Union[str, List[str]] = None) -> Table:
+                     freeze: Union[str, List[str]] = None, hide: Union[str, List[str]] = None,
+                     groups: List[dict] = None) -> Table:
```

Comment:
do we want to allow Union[dict, List[dict]] for groups?

Reply:
I think just the `List[dict]` is fine since the prop name implies there are multiple, but if you think we should allow both then I can switch it